### DISCUSSION TITLE
where to find the output files after plotFootprints function

### DISCUSSION THREAD

**Discussion by huangmiao1996** [GreenleafLab/ArchR#1851] [Questions / Documentation]
The full manual of ArchR 14.2 said "By default, these plots will be saved in the outputDirectory of the ArchRProject. If you requested to plot all motifs and returned this as a ggplot object, this ggplot object would be extremely large. An example of motif footprints from bias-subtracted analyses are shown below". However, I could not find the pictures or the outputDirectory. My ArchR version is 1.0.1. I hope that the question could be solved. Thank you!
<img width="793" alt="截屏2023-02-20 下午4 26 35" src="https://user-images.githubusercontent.com/110522446/220086356-963339dc-a68a-4666-84b2-31ada056dd57.png">

**Comment by rcorces** [comment]
Like many other plots, Footprint plots are saved in the `./Plots` subdirectory of your ArchRProject output directory. Your output directory is indicated by `getOutputDirectory(ArchRProj)`.

**Reply by rcorces** [reply]
Unless you dont provide an `ArchRProj` object to the `ArchRProj` parameter, in which case the output directory is relative to the current working directory.

**Reply by huangmiao1996** [reply]
Thank you! I have found it. By the way, ArchR is a powerful tool to analyze scATAC data.
